### ISSUE TITLE
`@aws-amplify/ui-vue` auth components stop working with `aws-amplify` 3.3.8+

### ISSUE THREAD

**Issue by neilgaietto** [aws-amplify/amplify-js#7703]
Reporting an error that I'm getting after upgrading `aws-amplify` and trying to use `<amplify-authenticator>` components found in `@aws-amplify/ui-vue`.

I have a vue app that had been working fine with the following packages:
- "aws-amplify": "^3.3.7",
- "@aws-amplify/ui-vue": "^0.2.26"

I upgraded to the following and started having issue:
- "aws-amplify": "^3.3.17",
- "@aws-amplify/ui-vue": "^0.3.1",
    
Before I show the error, here is what does work too (upgrade ui-vue, but not amplify):
- "aws-amplify": "^3.3.7",
- "@aws-amplify/ui-vue": "^0.3.1",
   

With the upgrade to anything after "aws-amplify": "^3.3.7" I get the following error when trying to use the `<amplify-authenticator>` component:
![image](https://user-images.githubusercontent.com/1541757/107134692-673e1c80-68c2-11eb-83a9-35f682995122.png)

I didn't change any configs because I saw no documentation about breaking changes. I should expect this minor update to not have breaking changes.

Its not an issue with the Vue components, just something in the aws-amplify package broke the ui components.

I am running this on a site that uses Quasar2.1.14/Vue2. I verified that this was not the issue as well. The original packages still worked with this.
I also verified that this happens only once I upgrade "aws-amplify" to anything from "^3.3.8" - "^3.3.17"

**Comment by wlee221** [comment]
Hello @neilgaietto! That error pops up whenever there's a mismatch between amplify versions. 

That being said, I've verified 

- "aws-amplify": "^3.3.17",
- "@aws-amplify/ui-vue": "^0.3.1"

to work locally on a Vue 2 app. I'm suspecting it's a lock file that's being a problem. Please try a fresh install with:
```bash
# yarn:
rm -rf node_modules yarn.lock && yarn 
# npm:
rm -rf node_modules package-lock.json && npm install
```
If the issue persists, please run
```bash
yarn list --pattern "@aws-amplify/auth" # yarn
npm list @aws-amplify/auth # npm 
```
and tell us what you get. Thanks!

**Comment by neilgaietto** [comment]
@wlee221 Thank you for the response. The package-lock.json was the issue.  During testing I installed the project on a fresh pc and still saw the issue, so I incorrectly assumed. 

So the question is, shouldn't running `npm update` be enough to get everything updated and versions in sync, or should deleting the lock file be part of the standard process? Because just using the update process has been the method I've used to get the latest package until now.
Apparently I should be reading up on this lock file more. Thanks!
